core.js

```diff
@@ -1326,6 +1326,13 @@ async function _fromTokenizer(tokenizer) {
 		};
 	}
 
+	if (checkString('ITSF')) {
```

Comment:
Please move this up to the 4-byte signatures, to prevent unnecessary data to be read.